### PR TITLE
feat(example): 2 new examples

### PR DESCRIPTION
**Pull request by Mte90**
I added my script that I use to analyze the shellcheck validation status in Amber using the tests as source and my lsp installer that I used as example for Amber usage (and shellcheck validation).

### CODE REVIEW

File: docs/by_example/lsp_installer.md
Comment (Ph0enixKM) [inline docs/by_example/lsp_installer.md:1]: For me this example script is pretty lengthy and doesn't highlight the core logic. Can you make a more condensed version that focuses on the most important Amber / stdlib features in action without much repetition?

**Review by Ph0enixKM — CHANGES_REQUESTED** [review]

**Review by Ph0enixKM — CHANGES_REQUESTED** [review]
Ah.. wrong button... meant to approve it instead

**Review by Ph0enixKM — APPROVED** [review]
It looks alright. Can anyone else review this? If noone has time to do it in the next 12h, you can merge it @Mte90

**Comment by Mte90** [comment]
@Ph0enixKM I can't merge it as requires 2 approvals...

File: docs/by_example/lsp_installer.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+# LSP installer
+
+This script automates the installation of several Language Server Protocol (LSP) tools, primarily by downloading them from GitHub and installing them on your system. The latest version of the script (and also **complete**) can be found [here](https://github.com/Mte90/My-Scripts/blob/master/dev/lsp-installer/install.ab).
+
+> The script uses standard library functions to simplify the process of downloading, unpacking, and installing various LSP tools. It checks for necessary permissions, downloads the latest releases of selected LSPs, moves them to system directories, makes them executable, and installs additional LSPs using `npm`, `pip`, and `gem` as needed.
```
[the remark this answers is not in the document]
Comment (Mte90) [reply docs/by_example/lsp_installer.md:5]: it is an information box for the website

**Pull request merged**